docker/sbtc/signer/signer-config.toml

```diff
@@ -26,7 +26,7 @@
 # Environment: SIGNER_EMILY__ENDPOINTS
 # Environment Example: '"https://1234567890abcdef@api.emilyexample.com",..'
 endpoints = [
-    "http://emily-server:3031",
+    "http://testApiKey@emily-server:3031",
```

Comment:
note that this api key will be able to trigger a reorg, so devenv will work slightly different from production where only the sidecar has this permission

Reply:
but we'll very soon remove this from the signer i guess, so maybe it doesn't matter too much right now.